covert/__main__.py

```diff
@@ -125,27 +125,49 @@ def argparse():
       args.files += args
       break
     if not a.startswith('--') and len(a) > 2:
-      stderr.write(f' 💣  Short arguments cannot be combined: covert {args.mode} {a}\n')
-      sys.exit(1)
+      shortargs = []
+      argvar = []
+      for x in ad.items():
+          for y in x[1]:
+              if y.startswith('--') and y.replace('--', '') in a:
+                  stderr.write(f' 💣  Multiple arguments cannot have long arguments: covert {args.mode} {a}\n')
+                  sys.exit(1)
+              elif y.startswith('-'):
+                  shortargs.append(y.replace('-',''))
+      for x in list(a[1:]):
+        if x not in shortargs:
+            stderr.write(f' 💣  {x} is not a short argument: covert {args.mode} {a}\n')
+            sys.exit(1)
+        argvar.append(next((k for k, v in ad.items() if f'-{x}' in v), None))
+        argvarlist = True
```

Comment:
Please indent by two spaces, not four. Set your editor to use indent width of 2 spaces (on this project) to do this automatically as you type in code.

I think the first `if` section here should be doing `continue`, rather than check for it being contained in the combined short argument. In principle you could have something like `-out` meaning `-o -u -t` which should not be rejected even though `--out` is also an argument.

Then this whole block could be written as a list comprehension like
```python
shortargs = [flag[1:] for switches in ad.values() for flag in switches if not flag.startswith("--")]
```
Although this is a bit complex already, so feel free to use regular loops if you prefer.